packages/babel-plugin-proposal-pipeline-operator/test/fixtures/smart/nested/input.js

```diff
@@ -1,5 +0,0 @@
-var result = 5
```

Comment:
Are these fixtures just moved to somewhere else? I cannot find them :thinking:

Reply:
Their removal was my mistake. I plan to restore them in a little bit.

Reply:
This was addressed in f9a1f1386de3bec4ad5e8a23a4695fe4db8f824d. 

The missing smart-mix tests were also duplicated to test Hack pipes. This uncovered a bug with Hack pipes (pipes within assignment expressions are not working), which I will work on later.